### ISSUE TITLE
Add field for tax number in invoice template

### ISSUE THREAD

**Issue by simonschaufi** [kimai/kimai#4174]
### Describe the problem

Whenever I move to another city, I get a new tax number which is given to me by the tax office. I would like to store this number in kimai instead of my template

### Describe the solution you'd like

Add another field for the tax number

**Comment by HeinzWuert** [comment]
plugin "custom fields" is the best solution for those additive fields

**Comment by simonschaufi** [comment]
@kevinpapst would you consider this a field to be added to the core or rather not?

**Comment by kevinpapst** [comment]
What is wrong with the field that already exists?
<img width="726" alt="Bildschirmfoto 2023-08-01 um 12 20 21" src="https://github.com/kimai/kimai/assets/533162/6563ed00-24a6-48a9-b4b5-6156af5f035a">

**Comment by simonschaufi** [comment]
No, that is not the tax number. The tax number is given to you by the Finanzamt and if you move to another city, it will change while the Umsatzsteuernummer will always be the same.
The Steuernummer also has a different Format.

Steuernummer und Umsatzsteuer-ID sind beides steuerliche Identifikationsnummern, über die das Finanzamt Rechnungen an Unternehmen zuordnen kann. Schreiben Sie eine Rechnung an einen Kunden in Deutschland, ist die Steuernummer auf der Rechnung ausreichend.

**Comment by kevinpapst** [comment]
Interesting, never heard about that requirements for invoices. I always only had my Umsatzsteuer-ID  on the invoice.
I guess you can never make a software fully compliant with all tax regulations out there (and that is only considering one country, let alone the rest of the world).

Can't you put it into one of the free text fields?

**Comment by simonschaufi** [comment]
Which free fields do you mean?

**Comment by MagicFab** [comment]
In Canada two fields are required to indicate such tax registration numbers, in addition to a label to name such numbers.
